Omnidir stereoCalibrate Assertion failed

Hi,
I have been trying to use the Omnidirectional camera model for stereo rectification, but even when just following the tutorial with the provided .xml I get a "Assertion failed".
The call that invokes this error is this one:
```
double rms = cv::omnidir::stereoCalibrate(objectPoints, imagePoints1, imagePoints2, imgSize1, imgSize2, K1, xi1, D1, K2, xi2, D2, rvec, tvec, rvecsL, tvecsL, flags, critia, idx);
```
The variables were set just like the tutorial on "modules/ccalib/tutorial/omnidir_tutorial.md" using the .xml on the "modules/ccalib/tutorial/data/"  folder.
The full error message is this:
```
OpenCV Error: Assertion failed (!fixedSize() || ((Mat*)obj)->size.operator()() == Size(_cols, _rows)) in create, file /var/tmp/portage/media-libs/opencv-3.1.0-r4/work/opencv-3.1.0/modules/core/src/matrix.cpp, line 2232
terminate called after throwing an instance of 'cv::Exception'
  what():  /var/tmp/portage/media-libs/opencv-3.1.0-r4/work/opencv-3.1.0/modules/core/src/matrix.cpp:2232: error: (-215) !fixedSize() || ((Mat*)obj)->size.operator()() == Size(_cols, _rows) in function create
```

I am using the opencv-3.1.0-r4 on a Gentoo.
Any ideas how to go around this error?

--- a/modules/ccalib/src/omnidir.cpp
+++ b/modules/ccalib/src/omnidir.cpp
@@ -1306,8 +1306,8 @@ double cv::omnidir::stereoCalibrate(InputOutputArrayOfArrays objectPoints, Input
     }
     if (om.empty())
     {
-        om.create(1, 3, CV_64F);
-        T.create(1, 3, CV_64F);
+        om.create(3, 1, CV_64F);
+        T.create(3, 1, CV_64F);
     }
     if (omL.empty())
     {

